web/public/js/directives/cd-action-bar/index.js

```diff
@@ -38,6 +45,16 @@
           $footer.off('inview');
           $element.remove();
         };
+
+        ctrl.$onChanges = function (changes) {
+          if (changes.forceFixed) {
```

Comment:
I guess it'll not trigger if changes.forceFixed isn't ?
It means you can't null it ?